app/frontend/src/components/autocomplete/autocomplete.js

```diff
@@ -1,42 +1,38 @@
 import accessibleAutocomplete from 'accessible-autocomplete';
 import 'accessible-autocomplete/dist/accessible-autocomplete.min.css';
+import { Controller } from '@hotwired/stimulus';
 import './autocomplete.scss';
+import api from '../../lib/api';
 
 const SHOW_SUGGESTIONS_THRESHOLD = 3;
 
-const highlightRefinement = (text, refinement) => {
+export const highlightRefinement = (text, refinement) => {
   const index = text.toLowerCase().indexOf(refinement.toLowerCase());
 
   /* eslint-disable max-len */
   return `${text.substring(0, index)}<span class='accessible-autocomplete__suggestion-highlight'>${text.substring(index, index + refinement.length)}</span>${text.substring(index + refinement.length, text.length)}`;
   /* eslint-enable */
 };
 
-const autocomplete = (fieldIds, source) => {
-  fieldIds.forEach((elementId) => {
-    const formInput = document.getElementById(elementId);
+export default class extends Controller {
+  connect() {
+    let currentInputValue = this.element.value;
```

Comment:
What's `this` in this context? (Still looking for candidates to be turned into targets)

Reply:
as just said on slack im going to remove autocomplete from this PR